lib/lte_link_control/lte_lc_shell.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <shell/shell.h>
+#include <modem/lte_lc.h>
+
+static int cmd_normal(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	int ret = lte_lc_normal();
+
+	if (ret) {
+		shell_error(shell, "Error: lte_lc_normal() returned %d", ret);
+	}
+
+	return 0;
```

Comment:
These functions could return `ret` instead. I'm not sure if the shell really uses the return value for anything, but would be good to anyway return a valid return value.

Reply:
Actually, we should return `-ENOEXEC`, so I fixed this a bit

```c
/**
 * @brief Shell command handler prototype.
 *
 * @param shell Shell instance.
 * @param argc  Arguments count.
 * @param argv  Arguments.
 *
 * @retval 0 Successful command execution.
 * @retval 1 Help printed and command not executed.
 * @retval -EINVAL Argument validation failed.
 * @retval -ENOEXEC Command not executed.
 */
typedef int (*shell_cmd_handler)(const struct shell *shell,
				 size_t argc, char **argv);
```